scripts/cuda/test_cuda_paths.py

```
@@ -0,0 +1,54 @@
+        print("No lib directory found in cuda_runtime package")
+        
+except ImportError:
+    print("nvidia.cuda_runtime not importable")
+
+# Try to find where PyTorch expects CUDA libraries
+print("\n=== PyTorch CUDA Library Expectations ===")
+import torch.utils.cpp_extension
+try:
+    cuda_home = torch.utils.cpp_extension.CUDA_HOME
+    print(f"PyTorch CUDA_HOME: {cuda_home}")
+except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)